apps/game_backend/lib/game_backend/items/consumable_item.ex

```diff
@@ -0,0 +1,29 @@
+defmodule GameBackend.Items.ConsumableItem do
+  @moduledoc """
+  ConsumableItem schema
+  """
+  use GameBackend.Schema
+  import Ecto.Changeset
+
+  alias GameBackend.Effects.ConfigurationEffect
+
+  @derive {Jason.Encoder, only: [:name, :radius, :effects, :mechanics]}
+  schema "consumable_items" do
+    field(:active, :boolean, default: false)
+    field(:name, :string)
+    field(:radius, :float)
+    field(:mechanics, {:map, :map})
+
+    has_many(:effects, ConfigurationEffect, on_replace: :delete)
```

Comment:
Why is this called ConfigurationEffect and not just Effect? Is this a temporary fix?
I think we could just use a list of strings (effect names) then create the effects table with the mechanics relation all at once, since the skills PR is already adding the Mechanic schema.

Reply:
Main reason is that already exists a schema named `effect.ex` but we can add this effect schema in another PR

Reply:
Yep. Also remember, we can name the schema Effect, but the table could be called "configurator_effects" to differentiate them in the DB.